docs/guide/blog.md

```diff
@@ -1,199 +1,14 @@
 ---
-order: 1
-parent:
-  order: 1
-  title: Getting Started with Starport
+title: "Module Basics: Blog"
+order: 3
 ---
 
-# Getting Started
+# Building a Blog
 
-## What is Starport?
+First, create a new blockchain:
```

Comment:
A description of what we want to achieve and how it's useful would be nice addition